src/components/create-appointment-fourth/styles.ts

```diff
@@ -0,0 +1,62 @@
+import { Avatar, FormControlLabel, ListItemText, Typography, styled } from '@mui/material';
+import { HEADER } from 'src/config-global';
+import { PRIMARY, SECONDARY } from 'src/theme/colors';
+import { TYPOGRAPHY } from 'src/theme/fonts';
+import typography from 'src/theme/typography';
+
+export const Background = styled('div')`
+  background-color: ${PRIMARY.light_main};
+  min-height: calc(100vh - ${HEADER.FLOW_HEIGHT}px - 1px);
+  padding: 24px;
+  gap: 24px;
+  display: flex;
+`;
+
+const AppointmentContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+  background-color: ${PRIMARY.white};
+  border-radius: 4px;
+  box-shadow: 0px 1px 16px 0px rgba(0, 0, 0, 0.1);
```

Comment:
could you please use consts or theme for colors?

Reply:
fixed